src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/ColorPickerTests/WinUIColorPickerPage.xaml.cs

```diff
@@ -0,0 +1,429 @@
+﻿using System;
+using Uno.UI.Samples.Controls;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Shapes;
+
+using ColorChangedEventArgs = Microsoft.UI.Xaml.Controls.ColorChangedEventArgs;
+using ColorPicker = Microsoft.UI.Xaml.Controls.ColorPicker;
+using ColorSpectrum = Microsoft.UI.Xaml.Controls.Primitives.ColorSpectrum;
+using ColorSpectrumComponents = Microsoft.UI.Xaml.Controls.ColorSpectrumComponents;
+using ColorSpectrumShape = Microsoft.UI.Xaml.Controls.ColorSpectrumShape;
+
+namespace UITests.Shared.Microsoft_UI_Xaml_Controls.ColorPickerTests
+{
+	[Sample("ColorPicker", "WinUI", Name="WinUIColorPickerPage")]
+    public sealed partial class WinUIColorPickerPage : UserControl
+    {
+        ToolTip colorNameToolTip;
+        Rectangle spectrumRectangle;
+        FrameworkElement colorSpectrumInputTarget;
+        Rectangle previousColorRectangle;
+        Panel selectionEllipsePanel;
+        Ellipse selectionEllipse;
+        ButtonBase moreButton;
+
+        public WinUIColorPickerPage()
+        {
+            this.InitializeComponent();
+
+            // Initialize the ColorPicker to a known default color so we have a known starting point.
+            this.ColorPicker.Color = Colors.Red;
+
+            this.ColorPicker.Loaded += ColorPicker_Loaded;
+
+            spectrumRectangle = null;
+            colorSpectrumInputTarget = null;
+            previousColorRectangle = null;
+            moreButton = null;
+
+            this.RedTextBlock.Text = this.ColorPicker.Color.R.ToString();
+            this.GreenTextBlock.Text = this.ColorPicker.Color.G.ToString();
+            this.BlueTextBlock.Text = this.ColorPicker.Color.B.ToString();
+            this.AlphaTextBlock.Text = this.ColorPicker.Color.A.ToString();
+
+            var verifySubclass = new MyColorSpectrum();
```

Comment:
Codacy found an issue: [Remove the unused local variable 'verifySubclass'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6121259)